test/app/controllers/albums.js

```diff
@@ -8,12 +8,15 @@ const chai = require('chai'),
   simple = require('simple-mock'),
   nock = require('nock'),
   fetch = require('node-fetch'),
+  MockDate = require('mockdate'),
   albums = require('./../../../app/models').album,
   User = require('./../../../app/models').user;
 
 const saltRounds = 10;
 
 beforeEach(() => {
+  MockDate.reset();
```

Comment:
this reset would be nice to have it in the `test/app.js` so anybody in the future can forget about it.